darwin/build.sh

```
@@ -0,0 +1,58 @@
+# setting the make TARGET based on selected device
+# and linking all executables to the same path, so we can use one build scheme
+case "$TARGET_PLATFORM_NAME" in
+  iphonesimulator)
+    export TARGET="IOS64SIM"
+    rm -f "$(pwd)"/darwin/XCSoarExecutable
+    ln -s "$(pwd)"/output/IOS64SIM/ipa/Payload/XCSoar.app "$(pwd)"/darwin/XCSoarExecutable
+    ;;
+  iphoneos)
+    export TARGET="IOS64"
+    rm -rf "$(pwd)"/output/IOS64/ipa "$(pwd)"/output/IOS64/xcsoar.ipa "$(pwd)"/output/IOS64/Info.plist*
+<<<<<<< HEAD
```

Shellcheck (0.10.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)